modules/acquia_cms_headless/src/Service/StarterkitNextjsService.php

```diff
@@ -160,9 +169,11 @@ public function createHeadlessConsumer(array $consumer_data) {
       $user = $this->getHeadlessUserData();
 
       if (!empty($user) && empty($consumers)) {
+
         $this->consumerSecret = $this->createHeadlessSecret();
         $consumer = Consumer::create();
         $consumer->set('label', $consumer_data['site-name']);
+        $consumer->set('client_id', $this->clientId);
```

Comment:
client ID should be machine name of label, example:
Label = Default Consumer
client ID = default_consumer

Reply:
As discussed this in DSM.. marking as resolved.